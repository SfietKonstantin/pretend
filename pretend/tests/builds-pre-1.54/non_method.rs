../build-sources/non_method.rs