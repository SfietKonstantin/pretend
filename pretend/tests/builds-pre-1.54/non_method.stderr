error: Failed to generate pretend implementation
 --> $DIR/non_method.rs:3:1
  |
3 | #[pretend]
  | ^^^^^^^^^^
  |
  = note: this error originates in an attribute macro (in Nightly builds, run with -Z macro-backtrace for more info)

error: Only methods are supported
 --> $DIR/non_method.rs:5:5
  |
5 |     type Item;
  |     ^^^^^^^^^^
